cms/envs/test.py

```diff
@@ -30,6 +30,8 @@
 from lms.envs.test import (
     WIKI_ENABLED,
     PLATFORM_NAME,
+    SITE_ID,
+    DEFAULT_SITE_ID,
```

Comment:
I'm not sure we need this setting anymore with the updates to edx/edx-django-sites-extensions#6 and edx/edx-platform#12555